packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -45,11 +45,12 @@ import { TIMELINE_ITEM_CONTEXT_MENU } from '@theia/timeline/lib/browser/timeline
 import { TimelineItem } from '@theia/timeline/lib/common/timeline-model';
 
 type CodeEditorWidget = EditorWidget | WebviewWidget;
-export namespace CodeEditorWidget {
-    export function is(arg: any): arg is CodeEditorWidget {
+@injectable()
+export class CodeEditorWidgetUtil {
+    is(arg: any): arg is CodeEditorWidget {
         return arg instanceof EditorWidget || arg instanceof WebviewWidget;
     }
-    export function getResourceUri(editor: CodeEditorWidget): CodeUri | undefined {
```

Comment:
minor: I'm not sure we need a breaking change note for this method, but I can certainly add a note tomorrow when I prepare the `changelog` for the release.